apps/admin-panel/lib/utils.ts

```diff
@@ -217,3 +217,27 @@ export const removeUnderscore = (str: string | undefined) => {
     .map((word) => word.charAt(0).toUpperCase() + word.slice(1))
     .join(" ")
 }
+
+/**
+ * Calls a function after {wait} time has passed
```

Comment:
Created this utility, I don't love this structure of having all utils in a single catch all file, and untested.